test/es-module/test-esm-loader-not-found.mjs

```diff
@@ -8,6 +8,7 @@ import { describe, it } from 'node:test';
 describe('ESM: nonexistent loader', () => {
   it('should throw', async () => {
     const { code, stderr } = await spawnPromisified(execPath, [
+      '--no-warnings',
```

Comment:
Why this change?
```suggestion
```

Reply:
because the test has nothing to do with warnings, so the warning just adds noise to the test output